examples/http-interceptor-error-handling/http_interceptor_error_handling.bal

```diff
@@ -1,20 +1,25 @@
 import ballerina/http;
-import ballerina/io;
 
-// Header name checked by the request interceptor.
-final string check_header = "checkHeader";
+type Album readonly & record {|
+    string title;
+    string artist;
+|};
 
-// Header value to be set to the request in the request error interceptor.
-final string request_check_header_value = "RequestErrorInterceptor";
+table<Album> key(title) albums = table [
+    {title: "Blue Train", artist: "John Coltrane"},
+    {title: "Sarah Vaughan and Clifford Brown", artist: "Sarah Vaughan"}
+];
 
 service class RequestInterceptor {
     *http:RequestInterceptor;
 
-    // This will return a `HeaderNotFoundError` if you do not set this header. 
+    // This will return a `HeaderNotFoundError` if you do not set the `x-api-version` header. 
     // Then, the execution will jump to the nearest `RequestErrorInterceptor`.
-    resource function 'default [string... path](http:RequestContext ctx,
-            @http:Header string checkHeader) returns http:NextService|error? {
-        io:println("Check Header Value : ", checkHeader);
+    resource function 'default [string... path](@http:Header string x\-api\-version, 
+            http:RequestContext ctx) returns http:NotImplemented|http:NextService|error? {
```

Comment:
Lets make the `http:RequestContext ctx` the first parameter.

Reply:
Lets do the below

```ballerina
resource function get albums(http:RequestContext ctx, @http:Header {name: "x-api-version"} string xApiVersion) 
        returns Album[] {
        // logic
}
```

Reply:
Fixed via [4d1c51a](https://github.com/ballerina-platform/ballerina-distribution/pull/3810/commits/4d1c51a5949bc06bfcadbe969b699cfa30c917c0)